packages/core/core/src/requests/PathRequest.js

```diff
@@ -62,11 +62,25 @@ async function run({input, api, options}: RunOpts) {
       options.autoinstall,
     ),
   });
-  let assetGroup = await resolverRunner.resolve(input);
+  let {
+    assetGroup,
+    filesUpdate,
+    filesCreation,
+    filesDeletion,
```

Comment:
Should we name these to match the below API? e.g. `invalidateOnFileUpdate`/`invalidateOnFileCreate`/`invalidateOnFileDelete`?

Reply:
These `invalidateOnFile*` function aren't a public API, so not necessarily 🤷‍♂️

Reply:
Personally I like being explicit that these fields are related to invalidation. Since these are not functions like the request api, a better naming pattern might be something like`fileUpdateInvalidations` and so on.